pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I run `mvn jar:jar` and `mvn test` on Fedora 25 and got an "Warning using system encoding".
The error wasn't present in either Ubuntu Jaunty (Travis CI) or Xenial (perhaps it was one of Fedora's "quirks")